cmake/ProjectWAVM.cmake

```diff
@@ -26,6 +26,10 @@ set(other_libraries ${platform_library} ${wasm_library} ${ir_library} ${logging_
 
 set(flags "-Wno-error -fvisibility=hidden")
 
+if(CMAKE_GENERATOR STREQUAL Ninja)
+    set(build_command BUILD_COMMAND ninja -j1)
```

Comment:
Any way to use the concurrency flag from cmake (if there's one)?

Reply:
Somehow this flag is not passed down to external projects for ninja. We can switch to make to make is simple.